implementing-nosto/implement-search/implement-search-using-api/implementing-category-pages.md

```diff
@@ -34,6 +37,10 @@ query {
 }
 ```
 
+{% hint style="info" %}
+Based on the provided and indexed fields, we can modify the requested fields included in the response. This adjustment, whether expanding or reducing the fields, is guided by the available options listed [here](https://search.nosto.com/v1/graphql?ref=SearchProduct).
```

Comment:
⛏️ This sentence is a bit complicated to understand. The reference to indexed fields is probably not necessary, because only extracted- and custom fields need to me marked as indexed (the admin UI marks standard product fields as indexed automatically, with no "official" way to exclude them). Given that the requester is fine with this, we can also keep it as it is.

Here is a simpler version, but feel free to ignore it:

```
Product fields that can be requested in the `hits` object are documented [here](https://search.nosto.com/v1/graphql?ref=SearchProduct). All indexed fields are accessible via the API.
```

Reply:
Your suggestion for sure is better any simplifying the info, applied!
Thanks!